content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -0,0 +1,91 @@
+* Build permissioned or nearly permissionless, centralized or largely decentralized ledgers whose assets can be traded on the Mainnet DEX
+* Create private or public sidechains, and possibly make the public sidechains available to the community to leverage various use cases. For example, a [smart sidechain with Hooks](https://hooks-testnet.xrpl-labs.com/) enabled.
+
+## How Do Federated Sidechains Work
+
+
+A sidechain is an independent ledger with its own consensus algorithm and transaction types and rules. Each sidechain runs its own set of servers (nodes and validators) and does not rely on the validators on the Mainnet to submit transactions on the sidechain.
+
+Each sidechain has two door accounts, one each on mainchain and sidechain, that are controlled by the federators on the sidechain.The federators independently listen for transactions on these door accounts.
+
+Federators, similar to validators on the mainchain but only live on the sidechain, jointly control the door accounts using the built-in multi-signature scheme available on the XRP Ledger. The door account has a signers list which consists of signing keys of all federators on the sidechain. 
+
+When a door account receives a transaction, the federator signs the transaction and broadcasts this information to other federators on the sidechain.
+Simultaneously, federators also listen for signed transactions from other federators and collect them.
+
+A transaction is valid only when 80% of the signers on the list have signed. A quorum is reached when 80% of federators have signed and the transaction is submitted to the door account and an asset is issued to the destination.
+
+When an asset is issued from the mainchain to the sidechain, that asset is locked up in the door account. The asset is only unlocked on the mainchain when a transaction from the sidechain to the main chain is submitted. Transactions within the sidechain are not visible to the servers on the sidechain. 
+
+
+## Terminology
+
+_Sidechain_: A sidechain is an independent ledger which provides a way to transfer assets from the XRP Ledger to the sidechain, and return assets from the sidechain back to the XRP Ledger. Each sidechain has its own consensus algorithm and transaction types and rules. 
+ 
+_Federator_: A server on the sidechain that listens for triggering transactions on both the main chain and the side chain. Each federator has a signing key associated with it that is used to sign transactions. A transaction must be signed by a quorum of federators before it can be submitted. Federators are responsible for creating and signing valid response transactions, collecting signatures from other federators, and submitting transactions to the main chain and side chain.
+
+_Main chain_: XRP Ledger where assets originate and where assets will be locked while being used on the side chain. For most applications, the main chain will be the XRP Ledger Devnet, Testnet, or Mainnet.
```

Comment:
```suggestion
_Mainchain_: The blockchain where assets originate and where assets are held while being used on the side chain. For most sidechains, the mainchain is the XRP Ledger Mainnet, Testnet, or Devnet.
```

- Use "mainchain" (one word) for consistency with the rest of the article
- Avoid future tense.